src/custom-elements/src/components/drawer/drawer.tsx

```diff
@@ -254,9 +254,13 @@ export class Drawer {
 
   render() {
     // TODO: change this into <chi-button/> element.
+    const xIconProperties = {icon: 'x', color: null};
```

Comment:
`color: null` should not be necessary. Please, remove it. instead, make the property color optional, that is what it is documented.